{"uuid":"733fc90a-bf72-4419-b759-125a003a8267","historyId":"8c8cecbfbfa743007bda86c42c9c9c61","fullName":"org.selenium.pom.tests.LoginTest.loginDuringCheckOut","labels":[{"name":"package","value":"org.selenium.pom.tests.LoginTest"},{"name":"testClass","value":"org.selenium.pom.tests.LoginTest"},{"name":"testMethod","value":"loginDuringCheckOut"},{"name":"parentSuite","value":"Suite1"},{"name":"suite","value":"Regression1"},{"name":"subSuite","value":"org.selenium.pom.tests.LoginTest"},{"name":"host","value":"Sunils-MacBook-Air.local"},{"name":"thread","value":"76104@Sunils-MacBook-Air.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginDuringCheckOut","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'Sunils-MacBook-Air.local', ip: '2401:4900:1c84:7f78:25c7:1dde:fd52:b2a6%en0'","trace":"java.lang.RuntimeException: org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'Sunils-MacBook-Air.local', ip: '2401:4900:1c84:7f78:25c7:1dde:fd52:b2a6%en0'\n\tat org.selenium.pom.factory.abstractFactory.RemoteDriverManagerFireFox.startDriver(RemoteDriverManagerFireFox.java:24)\n\tat org.selenium.pom.factory.abstractFactory.DriverManagerAbstract.getDriver(DriverManagerAbstract.java:25)\n\tat org.selenium.pom.base.BaseTest.startDriver(BaseTest.java:64)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:577)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\n\tat org.testng.TestNG.run(TestNG.java:1082)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'Sunils-MacBook-Air.local', ip: '2401:4900:1c84:7f78:25c7:1dde:fd52:b2a6%en0'\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:625)\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:241)\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:168)\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:148)\n\tat org.selenium.pom.factory.abstractFactory.RemoteDriverManagerFireFox.startDriver(RemoteDriverManagerFireFox.java:18)\n\t... 33 more\nCaused by: org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException\nBuild info: version: '4.15.0', revision: '1d14b5521b'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '15.0', java.version: '11.0.23'\nDriver info: driver.version: RemoteWebDriver\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:410)\n\tat org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:366)\n\tat org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:54)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:115)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:96)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:68)\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:163)\n\tat org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)\n\t... 37 more\nCaused by: java.util.concurrent.TimeoutException\n\tat java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1888)\n\tat java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2023)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:393)\n\t... 47 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"browser","value":"REMOTE_FIREFOX"}],"start":1730875120882,"stop":1730875120883}